persistent/ChangeLog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog for persistent
 
+## 2.14.6.3
+
+* [#1544](https://github.com/yesodweb/persistent/pull/1544)
+    * Fix type error message when no unique keys are defined on a model and you
+      use a function with constraint `AtLeastOneUniqueKey`.
```

Comment:
FWIW this popped up with another function I was using but it looks to come from the same place